sdk/communication/communication-network-traversal/test/public/communicationNetworkTraversalClient.spec.ts

```diff
@@ -78,48 +80,96 @@ matrix([[true, false]], async function (useAad) {
       }
     }).timeout(5000);
 
-    it("successfully gets a turn credential with identity and routeType any", async function () {
-      const connectionString = env.COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING;
-      const identityClient = new CommunicationIdentityClient(connectionString);
-      const user: CommunicationUserIdentifier = await identityClient.createUser();
-
-      const turnCredentialResponse = await client.getRelayConfiguration(user, "any");
+    it("successfully gets a turn credential providing roteType nearest", async function () {
+      const options: GetRelayConfigurationOptions = { routeType: "nearest" };
+      const turnCredentialResponse = await client.getRelayConfiguration(options);
       assert.isNotNull(turnCredentialResponse);
 
       const turnTokenExpiresOn = turnCredentialResponse.expiresOn;
       assert.isNotNull(turnTokenExpiresOn);
 
       const turnServers = turnCredentialResponse.iceServers;
+      assert.equal(turnServers.length, 1);
 
       for (const iceServer of turnServers) {
         for (const url of iceServer.urls) {
           assert.isNotNull(url);
         }
+        assert.isNotNull(iceServer.username);
+        assert.isNotNull(iceServer.credential);
+        assert.isNotNull(iceServer.routeType);
+        assert.equal(iceServer.routeType, "nearest");
+      }
+    }).timeout(5000);
+
+    it("successfully gets a turn credential providing roteType any", async function () {
+      const options: GetRelayConfigurationOptions = { routeType: "any" };
+      const turnCredentialResponse = await client.getRelayConfiguration(options);
+      assert.isNotNull(turnCredentialResponse);
+
+      const turnTokenExpiresOn = turnCredentialResponse.expiresOn;
+      assert.isNotNull(turnTokenExpiresOn);
 
+      const turnServers = turnCredentialResponse.iceServers;
+      assert.equal(turnServers.length, 1);
+
+      for (const iceServer of turnServers) {
+        for (const url of iceServer.urls) {
+          assert.isNotNull(url);
+        }
         assert.isNotNull(iceServer.username);
         assert.isNotNull(iceServer.credential);
         assert.isNotNull(iceServer.routeType);
         assert.equal(iceServer.routeType, "any");
       }
     }).timeout(5000);
 
-    it("successfully gets a turn credential with identity routeType nearest", async function () {
+    it("successfully gets a turn credential providing ttl", async function () {
+      const options: GetRelayConfigurationOptions = { ttl: 4000 };
+      const turnCredentialResponse = await client.getRelayConfiguration(options);
+      assert.isNotNull(turnCredentialResponse);
+
+      const turnTokenExpiresOn = turnCredentialResponse.expiresOn;
+      assert.isNotNull(turnTokenExpiresOn);
```

Comment:
Can we be more specific and assert that the expiry is less or equal than when the request was sent plus ttl?
We can store the now Date in line 129 before the request.